views/optin.inc.php

```diff
@@ -0,0 +1,92 @@
+<?php
+namespace Langchecker;
+
+use \Bugzilla\Bugzilla;
+
+$htmloutput = '<p id="back"><a href="http://l10n.mozilla-community.org/webdashboard/">Back to Web Dashboard</a></p>';
+
+// This view works only for mozilla.org (website ID 0)
+$current_website = $sites[0];
+$current_locale = $locale;
+
+if (! Project::isSupportedLocale($current_website, $current_locale)) {
+    $htmloutput .= "<p>This locale is not supported on mozilla.org</p>";
+    echo $htmloutput;
+    exit;
```

Comment:
These 3 lines could just be:
```php
exit($htmloutput . '<p>This locale is not supported on mozilla.org</p>');
```
(also, single quotes when there are no variable expansions)